src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/source/LinkedTransferQueue.class

```diff
@@ -0,0 +1,1749 @@
+/*
```

Comment:
I don't think this class is needed

Reply:
![image](https://user-images.githubusercontent.com/38923605/102529439-c07ca180-4104-11eb-981b-a2f410436597.png)

it is used in the Source Connector as its building up the internal buffer.

Reply:
But if it's importing from java.util.concurrent, why do we still need to make our own? I'm confused

Reply:
good point. i don't really know why this one was being used vs something from java.util.concurrent. i just know the class is being used. 

we can change the code to use native java data structures if these will suffice and give us the same functionality.